.expeditor/run_windows_tests.ps1

```diff
@@ -0,0 +1,15 @@
+# Stop script execution when a non-terminating error occurs
+$ErrorActionPreference = "Stop"
+
+# This will run ruby test on windows platform
+
+Write-Output "--- Bundle install"
+
+bundle config --local path vendor/bundle
```

Comment:
You can remove `exit $LASTEXITCODE` from the end of the script because the script will automatically exit with the last exit code just like a bash script.

However, a powershell script will not automatically fail if an executable command such as `bundle` fails so you need to include the following line after every line that runs an executable command to ensure that the powershell script sees the failure and exits appropriately.

```
If ($lastexitcode -ne 0) { Exit $lastexitcode }
```

Reply:
Added the failure condition